src/tests/cxxfrontend.c

```diff
@@ -253,23 +253,38 @@ void test_target()
 
 void test_emplace()
 {
-  Loc loc;
-  UnionExp ue;
-
-  IntegerExp::emplacei(&ue, loc, 1065353216, Type::tint32);
-  Expression *e = ue.exp();
-  assert(e->op == TOKint64);
-  assert(e->toInteger() == 1065353216);
-
-  UnionExp ure;
-  Expression *re = Compiler::paintAsType(&ure, e, Type::tfloat32);
-  assert(re->op == TOKfloat64);
-  assert(re->toReal() == CTFloat::one);
-
-  UnionExp uie;
-  Expression *ie = Compiler::paintAsType(&uie, re, Type::tint32);
-  assert(ie->op == TOKint64);
-  assert(ie->toInteger() == e->toInteger());
+    Loc loc;
+    UnionExp ue;
+
+    IntegerExp::emplacei(&ue, loc, 1065353216, Type::tint32);
+    Expression *e = ue.exp();
+    assert(e->op == TOKint64);
+    assert(e->toInteger() == 1065353216);
+
+    UnionExp ure;
+    Expression *re = Compiler::paintAsType(&ure, e, Type::tfloat32);
+    assert(re->op == TOKfloat64);
+    assert(re->toReal() == CTFloat::one);
+
+    UnionExp uie;
+    Expression *ie = Compiler::paintAsType(&uie, re, Type::tint32);
+    assert(ie->op == TOKint64);
+    assert(ie->toInteger() == e->toInteger());
+}
+
+/**********************************/
+
+void test_parameters()
```

Comment:
this isn't called from main

Reply:
Oops, you're right.  And there I thought everything worked just fine. :-)

Reply:
No worries, go ahead and merge it when its green.